server/etcdserver/api/etcdhttp/health.go

```diff
@@ -44,6 +44,7 @@ type ServerHealth interface {
 	Range(context.Context, *pb.RangeRequest) (*pb.RangeResponse, error)
 	Config() config.ServerConfig
 	AuthStore() auth.AuthStore
+	IsDefragActive() bool
```

Comment:
Please utilize https://github.com/etcd-io/etcd/pull/16836 instead of writing a separate mechanism to track defrag.

Reply:
I think it depends on if https://github.com/etcd-io/etcd/pull/16836 is backported to release-3.4 and release-3.5. Is there any objections on the backports? @ahrtr @serathius

Reply:
No any objection from my side.

We need to backport https://github.com/etcd-io/etcd/pull/16278 firstly.

Reply:
No objections from my side.